synapse/config/password.py

```diff
@@ -31,6 +31,10 @@ def read_config(self, config, **kwargs):
         self.password_localdb_enabled = password_config.get("localdb_enabled", True)
         self.password_pepper = password_config.get("pepper", "")
 
+        # Password policy
+        self.password_policy = password_config.get("policy", {})
+        self.password_policy_enabled = self.password_policy.pop("enabled", False)
```

Comment:
```suggestion
        self.password_policy_enabled = self.password_policy.get("enabled", False)
```

While unlikely, something else may try to may access this later.